x-pack/plugins/ml/public/components/anomalies_table/anomaly_details.js

```diff
@@ -188,12 +191,56 @@ export class AnomalyDetails extends Component {
     this.state = {
       showAllInfluencers: false
     };
+
+    this.tabs = [{
+      id: 'Details',
+      name: 'Details',
+      content: (
+        <Fragment>
+          <div className="ml-anomalies-table-details">
+            {this.renderDescription()}
+            <EuiSpacer size="m" />
+            {this.renderDetails()}
```

Comment:
now that we have a separate examples tab, should we remove the examples section from the details tab?

Reply:
I agree that the examples section in the details tab can go